web/components/logos/HalfScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+export default function HalfScreen() {
+    
+    return (
+        <div className="flex flex-col bg-custom-background w-full h-[20%] justify-center items-center sm:w-1/2 sm:h-full">
+            <div className="flex flex-row relative w-full h-[40%] justify-center sm:w-[17%] sm:h-[13%] sm:mb-[5%]">
+                <img src = '/MBBLogo.png' className="shrink-0 stroke-[2.1px] object-contain sm:stroke-[3px]"></img>
+            </div>
+            <div className="hidden sm:flex flex-col w-full h-[10%] justify-center shrink-0">
+                <p className="text-white text-center font-opensans text-[30px] font-bold leading-normal tracking-[0.9px] uppercase">
```

Comment:
Unless there are special cases, use Tailwind [font sizes](https://tailwindcss.com/docs/font-size) since they set both the font size and relative line height, so here instead of `text-[30px]` and `leading-normal`, you can just use `text-3xl`

Also, on Figma dev mode, you can change the code from CSS to Tailwind which is useful when it's not wrong so I'd still double-check it with the CSS and designs

Reply:
Also, for both screens, the text size should be `text-3xl sm:text-4xl` since it's larger on mobile, that was mb though since there were no designs for the loading page on Desktop 😅